src/Recorders/Concerns/LivewireRoutes.php

```diff
@@ -20,7 +20,7 @@ protected function resolveRoutePath(Request $request): array
         $path = $route->getDomain().Str::start($route->uri(), '/');
         $via = $route->getActionName();
 
-        if ($route->named('*livewire.update')) {
+        if ($route->named('*livewire.update') && $request->has('components.0.snapshot')) {
```

Comment:
Although I'm not against this change, could you let me know why you made it here?

Did you hit an issue?

Reply:
Hello, this changes comes to fix the problem commented [here](https://github.com/laravel/pulse/pull/340#issuecomment-2002682634). Was authored by @jessarcher [here](https://github.com/laravel/pulse/pull/340/commits/09ed4fb33c5bba560a41dec8bc7ac41c212b130e)

Reply:
Gotcha! Thanks.